core/metacling/src/TCling.cxx

```diff
@@ -5905,6 +5909,9 @@ static void* LazyFunctionCreatorAutoloadForModule(const std::string& mangled_nam
          if (!is_directory(DirPath))
             continue;
 
+         sPaths.push_back(Path);
+         int sPathsLast = sPaths.size() - 1;
```

Comment:
Can you move this next to the use?

Reply:
I just thought I should cache this instead of doing `sPaths.size() - 1` for each library